provisioning/service/src/ProvisioningServiceClient.cs

```diff
@@ -294,14 +296,13 @@ public Task<EnrollmentGroup> CreateOrUpdateEnrollmentGroupAsync(EnrollmentGroup
         /// This method will return the enrollment group information for the provided enrollmentGroupId. It will retrieve
         /// the correspondent enrollment group from the Device Provisioning Service, and return it in the
         /// <see cref="EnrollmentGroup"/> object.
-        ///
-        /// If the enrollmentGroupId does not exists, this method will throw <see cref="ProvisioningServiceClientException"/>.
         /// </remarks>
         /// <param name="enrollmentGroupId">The <c>string</c> that identifies the enrollmentGroup. It cannot be <c>null</c> or empty.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The <see cref="EnrollmentGroup"/> with the content of the enrollment group in the Provisioning Device Service.</returns>
-        /// <exception cref="ProvisioningServiceClientException">If the Provisioning Device Service was not able to retrieve
-        /// the enrollment group information for the provided enrollmentGroupId.</exception>
+        /// <exception cref="DeviceProvisioningServiceException">
+        /// If the service was not able to retrieve the enrollment group information for the provided enrollmentGroupId.
```

Comment:
Specify the parameter by ref.

```suggestion
        /// If the service was not able to retrieve the enrollment group information for the provided <paramref cref="enrollmentGroupId"/>.
```